app/controllers/users_controller.rb

```diff
@@ -0,0 +1,73 @@
+class UsersController < AdminController
+  before_action :set_user, only: %i[ show edit update destroy ]
+
+  def index
+    if params[:query].present?
+      @pagy, @users = pagy(User.global_search(params[:query]).order(created_at: :asc), items: 3)
+    else
+      @pagy, @users = pagy(User.order(created_at: :asc), items: 3)
+    end
+    respond_to do |format|
+      format.html
+      format.csv { send_data @users.to_csv, filename: "users-#{Date.today}.csv" }
+    end
+  end
+
+  def new
+    @user = User.new
+
+    respond_to do |format|
+      format.js
+      format.html 
+    end 
+  end
+  
+  def create
+    @user = User.new(user_params)
+    @user.skip_password_validation = true
+    @user.skip_confirmation!
+  
+    raw, enc = Devise.token_generator.generate(User, :reset_password_token)
+    @user.reset_password_token   = enc
+    @user.reset_password_sent_at = Time.now.utc
+
+    if @user.save
+      UserMailer.send_invite_email(@user,raw).deliver
+      redirect_to users_path
```

Comment:
you can write `redirect_to users_path, notice: "An invitation has been sent to the user!"`